src/elements/content-sidebar/versions/VersionsItem.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../common/variables';
+
+.bcs-VersionsItem {
+    background: transparent;
+    border: 1px solid transparent;
+    border-radius: 4px;
+    cursor: pointer;
+    display: flex;
+    font-size: 100%;
+    line-height: 20px;
+    padding: 15px 10px;
+    position: relative;
+    text-align: left;
+    width: 100%;
+
+    &.bcs-is-disabled {
+        cursor: default;
+        pointer-events: none;
+
+        .bcs-VersionsItem-badge {
+            background-color: $off-white;
+            color: $sevens;
+        }
+
+        .bcs-VersionsItem-info {
+            color: inherit;
+        }
+    }
+
+    &.bcs-is-selected {
+        border-color: $blue;
+    }
+}
+
+.bcs-VersionsItem-badge {
+    background-color: $light-blue;
+    border-radius: 4px;
+    color: $blue;
+    flex: 0 0 40px;
+    font-size: 18px;
+    margin-right: 10px;
+    padding: 10px 5px;
+    text-align: center;
+}
+
+.bcs-VersionsItem-details {
+    flex: 1 1 100%;
+    overflow: hidden;
```

Comment:
Is this overflow required?

Reply:
Yes, in order to support text overflow of its child components.